index.html

```diff
@@ -9142,6 +9161,252 @@
+          be fired on reception of an ACK from the remote side.</p>
+        </dd>
+        <dt><dfn><code>onclosed</code></dfn> of type <span class=
+        "idlAttrType"><a>EventHandler</a></span></dt>
+        <dd>
+          <p>This event handler, of event handler event type <code>closed</code>,
+          <em class="rfc2119" title="MUST">MUST</em> be fired when the stream is
+          closed by either side.</p>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Methods</h2>
+      <dl data-link-for="RTCQuicStream" data-dfn-for="RTCQuicStream" class=
+      "methods">
+        <dt><code>write</code></dt>
+        <dd>
+          <p><dfn>write()</dfn> writes the data to local buffer and send the data when the
+          transport underneath is writeable. If the buffer is full, it will fail.</p>
+          <table class = "parameters">
+            <tbody>
+              <tr>
+                <th>Parameter</th>
+                <th>Type</th>
+                <th>Nullable</th>
+                <th>Optional</th>
+                <th>Description</th>
+              </tr>
+              <tr>
+                <td class="prmName">data</td>
+                <td class="prmType"><code>Uint8Array</code></td>
+                <td class="prmNullFalse"><span role="img" aria-label=
+                "False">&#10008;</span></td>
+                <td class="prmOptFalse"><span role="img" aria-label=
+                "False">&#10008;</span></td>
+                <td class="prmDesc"></td>
+              </tr>
+            </tbody>
+          </table>
+          <div>
+            <em>Return type:</em> <code>void</code>
+          </div>
+        </dd>
+        <dt><code>finish</code></dt>
+        <dd>
+          <p><dfn>finish()</dfn> will maybe wirte one last value before finishing and then a "fin" bit.
+          <table class = "parameters">
+            <tbody>
+              <tr>
+                <th>Parameter</th>
+                <th>Type</th>
+                <th>Nullable</th>
+                <th>Optional</th>
+                <th>Description</th>
+              </tr>
+              <tr>
+                <td class="prmName">data</td>
+                <td class="prmType"><code>Uint8Array</code></td>
+                <td class="prmNullFalse"><span role="img" aria-label=
+                "False">&#10008;</span></td>
+                <td class="prmOptTrue"><span role="img" aria-label=
+                "True">&#10004;</span></td>
+                <td class="prmDesc"></td>
+              </tr>
+            </tbody>
+          </table>
+          <div>
+            <em>Return type:</em> <code>void</code>
+          </div>
+        </dd>
+        <dt><code>waitForWritable</code></dt>
+        <dd>
+          <p><dfn>waitForWritable()</dfn> will be resolved when the amount, up to maxBufferedAmount,
+          can be written or buffered. If the maxBufferedAmount is not provided, it will be equal
```

Comment:
It will be resolved when "amount" is available in the buffer, and the max buffered amount is maxBufferedAmount.   In other words, it waits until the buffer is drained if it's already too full.  That's not quite clear in this text as-is.